src/components/MenuBar/MenuBar.tsx

```diff
@@ -139,6 +141,11 @@ export default function MenuBar() {
 
         if (participantInformation && participantInformation.displayName !== '') {
           setParticipantInfo(participantInformation);
+          logger.push({
```

Comment:
please move to index.tsx (the highest one in the file hierarchy) if possible (and make sure it only gets called on first entry to the app). Also please try to send the entire navigator object (if it isn't too large (>1kb))